airbyte-integrations/bases/source-acceptance-test/setup.py

```diff
@@ -13,7 +13,7 @@
     "inflection~=0.5",
     "pdbpp~=0.10",
     "pydantic~=1.6",
-    "pytest~=6.1",
+    "pytest~=7.1.2",
```

Comment:
@airbytehq/api-connectors-dx can someone take a look in the impact of this change? Should we validate to all connectors?

Reply:
I added this change since running the trying to build failed using python 3.10. Reverted changes